src/app/app-routing.module.ts

```diff
@@ -13,6 +13,7 @@ import { SagaComponent } from './saga/saga.component';
 import { SugestoesComponent } from './sugestoes/sugestoes.component';
 import { TeatroComponent } from './teatro/teatro.component';
 import { VestibularesComponent } from './vestibulares/vestibulares.component';
+import { TecnologiaComponent } from './tecnologia/tecnologia.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/brunodiascultural/bookshelf_v1/pullRequest?prid=8812417)
```suggestion
import { TecnologiaComponent } from "./tecnologia/tecnologia.component";
```